Wikipedia/Code/ReadingListsController.swift

```diff
@@ -1,16 +1,20 @@
 import Foundation
 
-internal let WMFReadingListSyncStateKey = "WMFReadingListsSyncState"
-internal let WMFReadingListDefaultListEnabledKey = "WMFReadingListDefaultListEnabled"
-fileprivate let WMFReadingListSyncRemotelyEnabledKey = "WMFReadingListSyncRemotelyEnabled"
+// Sync keys
```

Comment:
Looks like all of these can remain restricted to WMF module via `internal`.

Reply:
Internal is the default access control so we don't need to add it explicitly https://developer.apple.com/swift/blog/?id=5